docs/pages/careers/developer-advocate.md

```diff
@@ -1,10 +1,10 @@
-# Developer Advocate
+# Developer Advocate / DevEx Engineer
```

Comment:
Instead?

Maybe 
```suggestion
# Developer Realtions Engineer
```

Reply:
Vercel uses the titles `Content Developer` and `Content Engineer` for roles that closely match what we're looking for here: someone with an engineering background who can build interesting demos and example projects to create learning content around.

Reply:
IMO, `Developer Relations Engineer` is probably the ideal title for this role, because I think it gets at the idea that this person needs to be a generalist in this area (they may be asked to perform all kinds of tasks associated with DevRel/DevEx - very wide scope)